text-ui-test/sources.txt

```diff
@@ -0,0 +1,11 @@
+../src/main/java/MissingTimeException.java
+../src/main/java/Response.java
+../src/main/java/Event.java
+../src/main/java/MissingDescriptionException.java
+../src/main/java/MissingNumberException.java
+../src/main/java/Task.java
+../src/main/java/Duke.java
+../src/main/java/Deadline.java
+../src/main/java/MissingDeadlineException.java
+../src/main/java/Todo.java
+../src/main/java/UnknownInputException.java
```

Comment:
You can write "../src/main/java/*.java" instead, to support extension to your code, so that you don't need to keep adding lines when creating new classes 😃